utils/helper.js

```diff
@@ -254,4 +256,63 @@ module.exports = {
       message: `User ${userName} uploaded using ${libraryName}`,
     });
   },
+  downloadFile: async (downloadUrl, localFilepath) => {
+    try {
+      const fileRes = await fetch(downloadUrl, {
+        method: "GET",
+        headers: new Headers({
+          "Content-Type": "application/pdf",
+        }),
+      });
+      const fileBuffer = await fileRes.buffer();
+      await fs.promises.writeFile(localFilepath, fileBuffer);
```

Comment:
Why are we writing the file locally? Can we not stream it directly like we do with Internet Archive?